addShirtAction.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+require_once 'ShirtsModel.php';
+
+$db = new PDO('mysql:host=db; dbname=collector_app', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$inputtedName = $_POST['name'] ?? false;
+$inputtedSeason = $_POST['season'] ?? false;
+$inputtedType = $_POST['type'] ?? false;
+$inputtedBrand_name = $_POST['brand_name'] ?? false;
+$inputted_img_url = $_POST['img_url'] ?? false;
+
```

Comment:
You're right in that there is no validation here. 

I do think it's a really important thing to add, I really want you to experience the 'pain' of manually writing validation before the see the magical way of doing in Laravel over the next couple of weeks.

You don't need to go nuts with it, some simple checks like making sure the length of the inputs are not too short/long will do the job.